.github/workflows/ci.yml

```
@@ -46,6 +47,8 @@ jobs:
 
     runs-on: ubuntu-latest
     name: Compile (py${{ matrix.python }}, root${{ matrix.root }})
+    steps:
+      - run: echo "Building with Python ${{ matrix.python }} and ROOT ${{ matrix.root }}"
 
     steps:
```

actionlint (1.7.7)

key "steps" is duplicated in "build" job. previously defined at line:50,col:5

(syntax-check)